classes/capitalize_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative './base_decorator'
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] [REQUIRED] Please change `upcase` to `capitalize` because similarly, `upcase` capitalizes every letter, and `capitalize` capitalizes the first letter of the string!

- [HINT]  Examples:

`"hello James!".upcase      #=> "HELLO JAMES!"`
`"hello James!".capitalize  #=> "Hello james!"`
`"hello James!".titleize    #=> "Hello James!" (Rails/ActiveSupport only)`